doc/plugin_agent_nodeattestor_azure_msi.md

```diff
@@ -2,22 +2,22 @@
 
 *Must be used in conjunction with the server-side azure_msi plugin*
 
-The `azure_msi` plugin attests nodes running in Microsoft Azure that have 
+The `azure_msi` plugin attests nodes running in Microsoft Azure that have
 Managed Service Identity (MSI) enabled. Agent nodes acquire a signed MSI token
 which is passed to the server. The server validates the signed MSI token and
 extracts the Tenant ID and Principal ID to form the agent SPIFFE ID. The SPIFFE
 ID has the form:
 
-```
-spiffe://<trust domain>/spire/agent/azure_msi/<tenant_id>/<principal_id>
+```xml
```

Comment:
This ain't XML!

Reply:
yeap, it is not, but I choose to put xml instead of text because it give use some interesting highlights, if you open the table now you can see that our examples has green color on "variables" [link](https://github.com/MarcosDY/spire/blob/add-md-lint/doc/plugin_agent_nodeattestor_azure_msi.md)